modules/developers-guide/pages/cli-reference/dfx-canister.adoc

```diff
@@ -669,6 +674,124 @@ Set "r7inp-6aaaa-aaaaa-aaabq-cai" as controller of "open_sf_assets".
 +
 You can now use the wallet canister `r7inp-6aaaa-aaaaa-aaabq-cai` to send cycles or add custodians to the `open_sf_assets` canister.
 
+== dfx canister send ==
+
+When creating your neuron stake, use the `+dfx canister send+` command with the `+dfx canister sign+` command when you want to separate these steps, rather than using the single `+dfx canister call+` command. Using separate calls can add security to the transaction.  
+
+=== Basic usage
+
+[source,bash,subs="quotes"]
+----
+dfx canister send _file_name_
+----
+
+=== Flags
+
+You can use the following optional flags with the `+dfx canister request-status+` command.
+
+[width="100%",cols="<32%,<68%",options="header"]
+|===
+|Flag |Description
+|`+-h+`, `+--help+` |Displays usage information.
+|`+-V+`, `+--version+` |Displays version information.
+|===
+
+=== Arguments
+
+You can specify the following argument for the `+dfx canister send+` command.
+
+[width="100%",cols="<32%,<68%",options="header"]
+|===
+|Argument |Description
+|`+file_name+` |Specifies the file name of the message.
+|===
+
+=== Examples
+
+Use the `+dfx canister send+` command to send a signed message created using the `+dfx canister sign+` command to the genesis token canister (GTC) to create a neuron on your behalf by running the following command:
+
+`+dfx canister send message.json+`
+
+== dfx canister sign ==
+
+When creating your neuron stake, use the `+dfx canister sign+` command with the `+dfx canister send+` command when you want to separate these steps, rather than using the single `+dfx canister call+` command. Using separate calls can add security to the transaction.
```

Comment:
```suggestion
Use the `+dfx canister sign+` command before sending a message with the `+dfx canister send+` command when you want to separate these steps, rather than using the single `+dfx canister call+` command. Using separate calls can add security to the transaction. 
For example, when creating your neuron stake, you might want to use the `+dfx canister sign+` command to create a signed `+message.json+` file using an air-gapped computer, then use the `+dfx canister send+` command to deliver the signed message from a computer connected to the network.
```

Create a neuron is one example, but you might also want to protect calls to start dissolving, disburse tokens, etc.